.travis.yml

```diff
@@ -0,0 +1,12 @@
+services:
+  - docker
+
+script:
+- echo "hello world"
+- docker build --target test --tag my-test-image .
+- docker run my-test-image todo_app/items_test.py    
+- docker run my-test-image todo_app/integration_test.py  
```

Comment:
This is fine here, but in a real project we'd expect new tests to be being added regularly, so combining these in e.g. a tests folder, and then just executing all of the tests in that folder might be more manageable